content/projects/gsoc/2020/projects/github-checks.adoc

```diff
@@ -27,53 +27,39 @@ By consuming this API, other plugins can easily create GitHub checks.
-* Consuming and implementation guide.
-* Final blog post and presentation slides.
-
-=== Architecture
-* The diagram below shows the class architecture:
-
-image:/images/post-images/gsoc-github-checks/architecture.png[title="GitHub Checks API Plugin Architecture" role="center" width=1000,height=500]
-
-=== Workflow
-* These properties are needed when creating a specific GitHub Checks:
-
-  ** APP identifier: Used to authenticate as a GitHub app in order to request access tokens for an installation (a user/organization account that installs the app on repositories) of  the app. 
-  This property is created by GitHub when registering Jenkins as a new GitHub App;
-
-  ** App installation id: Used to authenticate as an installation. We can easily get this property 
-  from link:https://developer.github.com/v3/activity/events/types/#installationrepositoriesevent[Installation Repository Event] 
-  when the users install the app on their GitHub repositories;
-
-  ** GitHub repository full name: Used to determine which repository the checks are making for. 
-  This property  can be retrieved from the scm API implemented by link:https://plugins.jenkins.io/github-branch-source/[GitHub Branch Source plugin];
-
-  ** Sha1: Used to determine which revision (a commit or pull request) the checks are created against. 
-  This property can also be retrieved from the scm API.
-
- * The diagram below shows the basic workflow for the GitHub Checks API plugin:
-
-image:/images/post-images/gsoc-github-checks/workflow.png[title="GitHub Checks API Plugin Workflow" role="center" width=1000,height=700]
-
-* More details can be found in the link:https://docs.google.com/document/d/1gpG7jeQmb9zz46t0ewNNRUrYbl7iHGScp5vgkn7MCXs/edit[original proposal].
+* A new Jenkins plugin (link:https://github.com/jenkinsci/checks-api-plugin[Checks API Plugin]) that defines the general checks API
+* A new Jenkins plugin (link:https://github.com/jenkinsci/github-checks-plugin[GitHub Checks Plugin]) that implements the geneal checks API for GitHub
+* A new feature (link:https://github.com/jenkinsci/warnings-ng-plugin/pull/550[pull request #550]) for plugin:warnings-ng[Warnings Next Generation] plugin to report warnings to GitHub by consuming the checks API
+* A new feature (link:https://github.com/jenkinsci/code-coverage-api-plugin/pull/169[pull request #169]) for plugin:code-coverage-api[Code Coverage API plugin] to report code coverage to GitHub by consuming the checks API
+* Pipeline support to allow users publishing customized checks
+* Rerun support on GitHub to allow users requesting a rebuild for failed builds
+
+=== Phase 1 Works
+.Demo [starts from 50:15]
+video::HQLhakpx5mk[youtube,width=800,height=420]
+
+* link:https://github.com/jenkinsci/checks-api-plugin[Checks API Plugin]
+* link:https://github.com/jenkinsci/github-checks-plugin[GitHub Checks Plugin]
+* link:/blog/2020/07/2020-07-09-github-checks-api-plugin-coding-phase-1.adoc[Blog post].
```

Comment:
I'll try to restore my slides for phase 1&2, since every time I just reused slides from the last phase and forget to store a copy...